crosswalks/Citation File Format 1.2.0.csv

```diff
@@ -1,76 +1,68 @@
-Property,Citation File Format Core (CFF-Core) 1.0.2
-codeRepository,repository-code
-programmingLanguage,
-runtimePlatform,
-targetProduct,
-applicationCategory,
-applicationSubCategory,
-downloadUrl,repository-artifact
-fileSize,
-installUrl,
-memoryRequirements,
-operatingSystem,
-permissions,
-processorRequirements,
-releaseNotes,
-softwareHelp,
-softwareRequirements,
-softwareVersion,
-storageRequirements,
-supportingData,
-author,authors
-citation,
-contributor,
-copyrightHolder,
-copyrightYear,
-dateCreated,
-dateModified,
-datePublished,date-released
-editor,
-encoding,
-fileFormat,
-funder,
-keywords,keywords
-license,license/license-url
-producer,
-provider,
-publisher,
-sponsor,
-version,version
-isAccessibleForFree,
-isPartOf,references
-hasPart,
-position,
-description,abstract
-identifier,doi
-name,title
-sameAs,
-url,url
-relatedLink,
-givenName,person.given-names
-familyName,person.name-particle + person.family-names + person.name-suffix
-email,person.email/entity.email
-affiliation,person.affiliation
-identifier,person.orcid / entity.orcid
-name,entity.name
-address,person.address + person.city + person.region + person.post-code + person.country / entity.address + entity.city + entity.region + entity.post-code + entity.country
-type,reference.type
-id,
-softwareSuggestions,
-maintainer,
-contIntegration,
-buildInstructions,
-developmentStatus,
-embargoDate,
-funding,
-issueTracker,repository
-referencePublication,references
-readme,
-,
-,
-,
-,
-,
-,
-,
-,
+Property,Citation File Format (1.2.0)
+codeRepository,repository-code
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,repository-artifact
+fileSize,
+installUrl,
```

Comment:
same set of url sourcesand problems as for `downloadUrl` above, I would suggest using `repository` here

Reply:
Resolved in 0e2f75ab1c9f52a3bf62d33100f55acb4c4469d3.